run-yarn-monitoring.py

```diff
@@ -1,16 +1,15 @@
-import requests
-from time import sleep
-from os import environ
-from datetime import datetime
 import sqlite3
+from datetime import datetime
+from os import environ
+from time import sleep
 
-# Connection to NM database needed to retrieve total cpus of each node
+import requests
+
+# The nodemanager database contains the number of logical cores of each NM
 filename_nmdb = environ['COBALD_TARDIS_NODEMANAGER_DATABASE']
-nmconn = sqlite3.connect(filename_nmdb)
-nmcursor = nmconn.cursor()
-cpus_query = "SELECT cpus FROM yarn_nm WHERE name = ?"
+cpus_query = 'SELECT cpus FROM yarn_nm WHERE name = ?'
 
-class YarnResourceManager:
+class YarnResourceManager(object):
```

Comment:
Why `(object)`? In python3 it shouldn't matter.

Reply:
Ah yes, true. Just an habit from writing python2 compatible code